django_cloud_deploy/integration_tests/workflow_test.py

```diff
@@ -246,3 +248,49 @@ def test_deploy_new_app_sync(self):
                     secrets=secrets)
                 response = requests.get(admin_url)
                 self.assertIn('Django administration', response.text)
+
+
+class ProjectWorkflowIntegrationTest(test_base.BaseTest):
+    """Integration test for django_cloud_deploy.workflow._project."""
+
+    def setUp(self):
+        super().setUp()
+        self._project_workflow = _project.ProjectWorkflow(self.credentials)
+
+    def test_create_new_project_no_permission(self):
+        project_name = 'New Project'
+        project_id = utils.get_resource_name(resource_type='project')
+
+        exception_regex = r'.*HttpError 403.*'
+
+        # The provided credentials object does not have permission to create
```

Comment:
Why does this credential not work but in the other tests it does work?

Reply:
This credentials object only allow users to manage resources under a pre-created GCP project. Other tests can do whatever they need to do under that project, but they do not have access to other projects, including creation of other projects.